app/code/community/Reverb/ReverbSync/sql/reverbsync_setup/mysql4-upgrade-0.1.6-0.1.7.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
i think the version number in this file is now out of date with other migrations. can I just change the file name safely?